packages/swapper/src/swappers/RelaySwapper/utils/types.ts

```diff
@@ -0,0 +1,237 @@
+  inTxHashes: string[]
+  txHashes: string[]
+  time: number
+  originChainId: number
+  destinationChainId: number
+}
+
+export type AppFee = {
+  recipient: string
+  fee: string
+}
+
+export type Transaction = {
+  to: string
+  value: string
+  data: string
+}
+
+export type QuoteParams<T extends 'quote' | 'rate'> = {
+  user: T extends 'quote' ? string : undefined
+  originChainId: number
+  destinationChainId: number
+  originCurrency: string
+  destinationCurrency: string
+  tradeType: 'EXACT_INPUT' | 'EXACT_OUTPUT' | 'EXPECTED_OUTPUT'
+  recipient?: string
+  amount?: string
+  txs?: Transaction[]
+  referrer?: string
+  refundTo?: string
+  refundOnOrigin?: boolean
+  useReceiver?: boolean
+  useExternalLiquidity?: boolean
+  usePermit?: boolean
+  useDepositAddress?: boolean
+  slippageTolerance?: string
+  appFees?: AppFee[]
+  gasLimitForDepositSpecifiedTxs?: number
+  userOperationGasOverhead?: number
+  forceSolverExecution?: boolean
+}
+
+export type RelayToken = {
+  chainId: number
+  address: string
+  symbol: string
+  name: string
+  decimals: number
+  metadata: {
+    logoURI: string
+    verified: boolean
+    isNative: boolean
+  }
+}
+
+export type CallFees = {
+  gas: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayer: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayerGas: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayerService: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  app: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+}
+
+export type QuoteDetails = {
+  currencyIn: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  currencyOut: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  totalImpact: {
+    usd: string
+    percent: string
+  }
+  rate: string
+  slippageTolerance: {
+    origin: {
+      usd: string
+      value: string
+      percent: string
+    }
+    destination: {
+      usd: string
+      value: string
+      percent: string
+    }
+  }
+  timeEstimate: number
+  userBalance: string
+}
+
+// https://github.com/reservoirprotocol/relay-kit/blob/main/packages/sdk/src/types/Execute.ts
+export type Execute = {
+  errors?: {
+    message?: string
+    orderId?: string
+  }[]
+  fees: CallFees
+  details: QuoteDetails
+  error?: any
+  refunded?: boolean
+  steps: {
+    error?: string
+    errorData?: any
+    action: string
```

Comment:
Remove me, we don't consume this

Reply:
Removed in [9535276](https://github.com/shapeshift/web/pull/9188/commits/95352767db00b2aaec3c108b3cacfcabd09b6e4c)